frontend/src/lib/components/DefinitionPopup/DefinitionPopup.tsx

```diff
@@ -441,6 +453,7 @@ export const renderItemPopup = (
                 <DefinitionPopup.HorizontalLine />
                 {/* Things start to get different here */}
                 {renderRestOfDefinition(item, listGroupType, hasTaxonomyFeatures)}
+                <DefinitionPopup.Type item={item as PropertyDefinition} />
```

Comment:
Can we place this inside of the renderRestOfDefinition method? I think it'd be cleaner to render this only when item is a property definition.